.github/workflows/tagger.yml

```diff
@@ -17,27 +17,22 @@ jobs:
 
       - name: Check current version
         id: check_version
-        run: echo "VERSION=$(node -p "require('./packages/core/package.json').version")" >> $GITHUB_OUTPUT
-
-      - name: Check remote tags
-        id: check_tags
-        run: |
-          git fetch --tags
-          echo "TAGS=$(git tag -l)" >> $GITHUB_OUTPUT
+        run: echo "TAG=v$(node -p "require('./packages/core/package.json').version")" >> $GITHUB_OUTPUT
 
       - name: Check if tag exists
         id: check_tag
         run: |
-          if [[ "${{ steps.check_tags.outputs.TAGS }}" == *"${{ steps.check_version.outputs.VERSION }}"* ]]; then
-            echo "TAG_EXISTS=true" >> $GITHUB_OUTPUT
-          else
+          git fetch --tags
+          if [ -z "$(git tag -l | grep -w ${{ steps.check_version.outputs.TAG }})" ]; then
```

Comment:
Edge case: a.b.c matches a.b.c-alpha.0?

Reply:
You're right, I make a simple test:

```bash
> echo -e "a.b.c-alpha.0" | grep -w a.b.c
a.b.c-alpha.0
```

But I don't know how to construct a full match in `grep`. Anyone aware?

Reply:
`^` & `$`?

Reply:
Also `$` should be escaped as `\$`, yes.